### DISCUSSION TITLE
Next JS 13 with both Python and Node.js API routes

### DISCUSSION THREAD

**Discussion by avalanche-tm** [vercel/next.js#58647] [Help] [answered]
### Summary

Is there any working example of NextJS 13 with both Python and Node.js API routes that works both locally and when deployed to Vercel?

Examples like this one https://vercel.com/templates/python/flask-hello-world do not show how to use **mix of Javascript and Python** API functions. They assume you want to completely replace your backend with Python.

Thanks in advance

**Comment by leerob** [comment, marked as answer]
Hey there! We have a template here with Next.js and a Flask API: https://vercel.com/templates/next.js/nextjs-flask-starter

**Reply by EliotKhachi** [reply]
Hey, I have the same question as avalanche-tm, and I'd appreciate some clarification / confirmation.

So does the NextJS Flask API template map the`/api/python` to the flask server, and I can configure routes under that, i.e. `/api/python/hello`? Would my other routes written in javascript, i.e. `/api/auth`, continue to run serverless on nodeJS through vercel?

Also, when I go to the `/api/python` endpoint on the production build of the template, I get a 500 internal server error. Not sure if I'm doing something wrong.
`https://nextjs-flask-starter-orpin-nine.vercel.app/api/python`

Thanks,
Eliot
